chaco/colormapped_scatterplot.py

```diff
@@ -434,7 +427,7 @@ def draw_func(x, y, size):
                     draw_func(x[i], y[i], size)
 
             else:
-                path = marker_cls.custom_symbol
+                path = self.custom_symbol
```

Comment:
`custom_symbol` is a trait on the `ColormappedScatterPlot`, not the marker. For example use, see test `test_scatter_custom`.